descriptors/ModuleDescriptor-template.json

```diff
@@ -234,7 +234,7 @@
     },
     {
       "id": "patron-notice-policy-storage",
-      "version": "0.10",
+      "version": "1.0",
```

Comment:
Why is this changing to version 1.0? That would suggest the API is considered stable. My understanding is that it is still experimental

Reply:
Seems like there is no reason for this. We'll change it to `0.11`

Reply:
The version has been updated to 0.11